packages/components/src/templates/next/components/internal/PaginationControls/PaginationControls.tsx

```diff
@@ -25,12 +27,14 @@ export function PaginationControls({
   setCurrPage,
   onPageChange,
 }: PaginationControlsProps) {
+  const isTablet = useMediaQuery("(min-width: 768px)")
```

Comment:
nit: I think we should probably centralise this breakpoint?

Reply:
ya maybe i take from tailwind config. let me update.

Reply:
added a `useBreakpoint` hook, but cannot get from tailwind config since others may override the breakpoints and it won't match liow. I just hardcode first if that's the case anyways